scripts/components/LviFavorites.ts

```diff
@@ -21,11 +22,35 @@ export default class LviFavorites extends LviFavoritesDesign {
     set itemTitle(value: string) {
         this.lblFavoriteItemTitle.text = value;
     }
-    get itemPrice(): any {
-        return this.lblFavoriteItemPrice.text;
+    get itemPrice(): string {
+        return this.tvPrice.text;
+    }
+    set itemPrice(value: string) {
+        const discountExists = !!this.tvDiscount.text;
+        let attributeString = new AttributedString({
+            strikethrough: discountExists,
+            ios: {
+                strikethroughColor: themeService.getNativeStyle('.product-price').textColor
+            },
+            string: value || '',
+            font: themeService.getNativeStyle(discountExists ? '.product-price.discount' : '.product-price.nodiscount').font,
+            foregroundColor: themeService.getNativeStyle(discountExists ? '.product-price.discount' : '.product-price.nodiscount').textColor
+        });
+        this.tvPrice.scrollEnabled = false;
+        this.tvPrice.attributedText = [attributeString];
+    }
+    get itemDiscount(): string {
+        return this.tvDiscount.text;
     }
-    set itemPrice(value: any) {
-        this.lblFavoriteItemPrice.text = value;
+    set itemDiscount(value: string) {
+        let attributeString = new AttributedString({
```

Comment:
We should use const if the variable will not change in the next lines.

Reply:
DONE.